src/aspire/abinitio/commonline_c3_c4.py

```diff
@@ -680,16 +682,16 @@ def _J_sync_power_method(self, vijs):
         itr = 0
 
         # Power method iterations
+        logger.info("Initiating Power Method.")
```

Comment:
geometric... so for any ratio we can compute the estimated steps remaining with arithmetic? like maybe log(eps) / log(lamb2 /lamb1) or something to that effect?

Reply:
Maybe something like, but in the current form we're not even calculating the eigenvalues so this would take some work. It involves computing the Rayleigh quotient, which would be done on-the-fly in our case. It would also involve finding the top two eigenvectors instead of just the one we're finding now.

Reply:
Ah, okay, the way you explained it sounded like you already had things handy. 

Yeah, no point slowing it down more.  Thanks, hope it was more interesting than bug fixing at least :)